relay-server/src/metrics_extraction/transactions.rs

```diff
@@ -414,9 +415,18 @@ fn extract_transaction_metrics_inner(
                 tags_for_measurement.insert("measurement_rating".to_owned(), rating);
             }
 
+            let metric_name = format!("measurements.{}", name);
```

Comment:
This is the line which makes this PR necessary in the first place: The metric name is generated on-the-fly from whatever measurement name the event contains, so it might be invalid.